app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -10,6 +10,7 @@ def new
 
     def create
       @form = ProviderUserForm.new(provider_user_params.merge(available_providers: Provider.all))
+      @form.dfe_sign_in_uid = nil if @form.dfe_sign_in_uid.blank?
```

Comment:
It might be safer to change the getter on `ProviderUserForm` to return nil if this is blank

Reply:
I've added validation to ProviderUser to allow nil but ensure the uid is not set to whitespace. L13 above now merely makes sure the resulting model is valid.